lms/envs/aws.py

```diff
@@ -424,6 +424,11 @@
 
 GRADES_DOWNLOAD = ENV_TOKENS.get("GRADES_DOWNLOAD", GRADES_DOWNLOAD)
 
+###### CORS ###########
+
+CORS_ORIGIN_WHITELIST = AUTH_TOKENS.get('CORS_ORIGIN_WHITELIST', CORS_ORIGIN_WHITELIST)
+CORS_ORIGIN_ALLOW_ALL = AUTH_TOKENS.get('CORS_ORIGIN_ALLOW_ALL', CORS_ORIGIN_ALLOW_ALL)
```

Comment:
Note for reviewers: there's also a `CORS_ORIGIN_REGEX_WHITELIST` setting we could expose.  Is that something we'd want, given that we currently have a small number of domains?